packages/edit-post/src/components/secondary-sidebar/list-view-sidebar.js

```diff
@@ -109,62 +115,51 @@ export default function ListViewSidebar( { listViewToggleElement } ) {
 		handleToggleListViewShortcut
 	);
 
-	/**
-	 * Render tab content for a given tab name.
-	 *
-	 * @param {string} tabName The name of the tab to render.
-	 */
-	function renderTabContent( tabName ) {
-		if ( tabName === 'list-view' ) {
-			return (
-				<div className="edit-post-editor__list-view-panel-content">
-					<ListView dropZoneElement={ dropZoneElement } />
-				</div>
-			);
-		}
-		return <ListViewOutline />;
-	}
-
 	return (
 		// eslint-disable-next-line jsx-a11y/no-static-element-interactions
 		<div
 			className="edit-post-editor__document-overview-panel"
 			onKeyDown={ closeOnEscape }
 			ref={ sidebarRef }
 		>
-			<Button
-				className="edit-post-editor__document-overview-panel__close-button"
-				icon={ closeSmall }
-				label={ __( 'Close' ) }
-				onClick={ closeListView }
-			/>
-			<TabPanel
+			<div
 				className="edit-post-editor__document-overview-panel__tab-panel"
 				ref={ tabPanelRef }
-				onSelect={ ( tabName ) => setTab( tabName ) }
-				selectOnMove={ false }
-				tabs={ [
-					{
-						name: 'list-view',
-						title: _x( 'List View', 'Post overview' ),
-						className: 'edit-post-sidebar__panel-tab',
-					},
-					{
-						name: 'outline',
-						title: _x( 'Outline', 'Post overview' ),
-						className: 'edit-post-sidebar__panel-tab',
-					},
-				] }
 			>
-				{ ( currentTab ) => (
+				<Tabs
+					selectOnMove={ false }
```

Comment:
Echoing @andrewhayward 's question — is there a reason for this setting?

Reply:
My reason for including it was to maintain parity with existing behavior, but after thinking about this question I wish I'd looked a little more into why it was previously set up this way under `TabPanel`.

Disabling automatic selection was [first introduced](https://github.com/WordPress/gutenberg/pull/50199#discussion_r1181006346) when `TabPanel` was implemented. It was used to [avoid the sidebar's focus behavior from switching tabs in ways that would be disruptive or unexpected for users](https://github.com/WordPress/gutenberg/pull/50199#issuecomment-1568472931).

All of that said, no matter how I test it I can't actually replicate a case where I'm looking at one tab, and the focus behavior drags me to the other. If I select the **Outline** tab, then focus the canvas, and then hit the shortcut keys, focus goes to my current tab, which is **Outline**. If focus went to **List View** and it were automatically selected, that could be a little disorienting, but it doesn't seem to be possible.

Theory: The implementation of `TabPanel` in this sidebar predates that component getting updated with `Ariakit` internals. The reason focus lands on the currently selected tab is because of how `Ariakit` applies a negative `tabindex` to all tabs that aren't currently selected.

I suspect pre-`Ariakit` `TabPanel` didn't have a similar implementation, meaning the first tab, even unselected, could have received focus. Without the prop in question, that would cause a potentially jarring change. Rolling back to the old `TabPanel` without `Ariakit` is kind of a pain on my setup, so I'm going to roll with this assumption being accurate.

So to sum up, I think this was a necessary prop at the time, but since the arrival of `Ariakit` we probably no longer need it. Thanks for asking the question I should have asked myself at the outset 😉